packages/svelte-ux/src/lib/components/DateField.svelte

```diff
@@ -21,6 +22,13 @@
   $: actualFormat = format ?? $format_ux.settings.formats.dates.baseParsing ?? 'MM/dd/yyyy';
   $: actualMask = mask ?? actualFormat.toLowerCase();
 
+  type FieldClasses = NonNullable<ComponentProps<Field>['classes']>;
+
+  export let classes: {
+    root?: FieldClasses['root'];
```

Comment:
This can just be `string` in this case

Reply:
Yep! I'll replace that and remove `NonNullable`

Reply:
Resolved in [209b2c3](https://github.com/techniq/svelte-ux/pull/244/commits/209b2c393bdfd142f1b5e902f9b618350ffaf7bb)